src/main/java/me/leoko/advancedban/Universal.java

```diff
@@ -69,11 +77,8 @@ public void setup(MethodInterface mi) {
         mi.setupMetrics();
         PunishmentManager.get().setup();
 
-        for (Command command : Command.values()) {
-            for (String commandName : command.getNames()) {
-                mi.setCommandExecutor(commandName, command.getTabCompleter());
-            }
-        }
+        Arrays.stream(Command.values()).forEach(command -> Arrays.asList(command.getNames())
+                .forEach(commandName -> mi.setCommandExecutor(commandName, command.getTabCompleter())));
```

Comment:
This one got a bit messy. I think it's best to keep using for-each loops for Arrays